lib/plugins/cannulaage.js

```diff
@@ -16,6 +16,7 @@ function init() {
     var age = 0;
     var found = false;
     var treatmentDate = null;
+    var message = 'no notes';
```

Comment:
maybe instead of showing "no notes" we can just push 1 more label/value to the info array if there is a note